.github/workflows/python-app.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         sudo apt install -y libffi-dev libssl-dev python3-pip xmlsec1 procps libpcre3 libpcre3-dev
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip
+        ## python -m pip install --upgrade pip
```

Comment:
no, please, do the update here

Reply:
did you mean remove the line 35 (commented)?

Reply:
yes

Reply:
done